ReplayRequests/ReplayRequests/Program.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Microsoft.ServiceBus.Messaging;
+using Microsoft.WindowsAzure;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace ReplayRequests
+{
+    class Program
+    {  
+        public static void Main(string[] args)
+        {
+            try
+            {
+                using (var requestsFile = File.CreateText("Rquests.txt"))
+                {
+                    var connectionString = CloudConfigurationManager.GetSetting("Microsoft.ServiceBus.ConnectionString");
+                    var queueClient = QueueClient.CreateFromConnectionString(connectionString, CloudConfigurationManager.GetSetting("QueuePath"));
+                    var cancellationTokenSource = new CancellationTokenSource();
+                    var cancellationToken = cancellationTokenSource.Token;
+                    var task = ContinouslyReadFromQueue(cancellationToken, queueClient, requestsFile);
+                    Console.WriteLine("Press any key to exit...");
+                    Console.ReadKey();
+                    cancellationTokenSource.Cancel();
+                    task.Wait();
+                }
+            }catch(Exception e)
+            {
+                Console.WriteLine("Exception message : {0} {1} Stack trace : {2}", e.Message, Environment.NewLine, e.StackTrace);
+            }
+        }
+
+        private static async Task ContinouslyReadFromQueue(CancellationToken cancellationToken, QueueClient queueClient, TextWriter requestsFile)
```

Comment:
This looks like a single threaded, and in the foreground type work. Where you instead probably want to run it on multiple threads to saturate the network (or at least mimic a similar load in production).

Do you plan to run it on its own VM?